api-catalog-package/src/main/resources/bin/start.sh

```diff
@@ -85,6 +97,6 @@ _BPX_JOBNAME=${ZOWE_PREFIX}${CATALOG_CODE} java \
     -Djava.protocol.handler.pkgs=com.ibm.crypto.provider \
     -Dloader.path=${COMMON_LIB} \
     -jar "${JAR_FILE}" &
-pid=$?
-
+pid=$!
+echo "pid=${pid}"
```

Comment:
I guess this one is intentional as it is used by other scripts?

Reply:
It's in all three scripts.

Reply:
this one is intentional, we need to store this PID as it is not available in java unix process object